README-local.md

```diff
@@ -0,0 +1,298 @@
+
+    error_page 404 /index.php;
+
+    location ~ \.php$ {
+        fastcgi_pass unix:/var/run/php/php7.2-fpm.sock;
+        fastcgi_param SCRIPT_FILENAME $realpath_root$fastcgi_script_name;
+        include fastcgi_params;
+    }
+
+    location ~ /\.(?!well-known).* {
+        deny all;
+    }
+}
+```
+
+Restart the nginx server
+
+```
+sudo service nginx restart
+```
+
+## Install npm
+
+*For more detailed instructions, please visit [here](https://linuxize.com/post/how-to-install-node-js-on-ubuntu-18.04/)*
+
+
+```
+curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+sudo apt install -y nodejs
+```
+
+## Install composer
+
+```
+sudo apt-get install composer
+```
+
+## Download Laravel Installer
+
+```
+composer global require "laravel/installer=~1.1"
+sudo apt install php-mbstring php-xml php-bcmath
+```
+
+## Run Talent Cloud
+
+### Clone repo
+
+#### HTTPS
+
+```
+git clone  https://github.com/GCTC-NTGC/TalentCloud.git
+```
+
+#### SSH
+
+```
+git clone git@github.com:GCTC-NTGC/TalentCloud.git
+```
+
+### Setup self signed cert
+
+*For more details, please visit [here](https://webscoot.io/blog/create-self-signed-certificate-ubuntu-windows-nginx/)*
+
+Paste the below config into sudo vi localhost.conf
+
+```
+[req]
+default_bits       = 2048
+default_keyfile    = localhost.key
+distinguished_name = req_distinguished_name
+req_extensions     = req_ext
+x509_extensions    = v3_ca
+
+[req_distinguished_name]
+countryName                 = Country Name (2 letter code)
+countryName_default         = CA
+stateOrProvinceName         = State or Province Name (full name)
+stateOrProvinceName_default = Ontario
+localityName                = Locality Name (eg, city)
+localityName_default        = Ottawa
+organizationName            = Organization Name (eg, company)
+organizationName_default    = localhost
+organizationalUnitName      = organizationalunit
+organizationalUnitName_default = Development
+commonName                  = Common Name (e.g. server FQDN or YOUR name)
+commonName_default          = localhost
+commonName_max              = 64
+
+[req_ext]
+subjectAltName = @alt_names
+
+[v3_ca]
+subjectAltName = @alt_names
+
+[alt_names]
+DNS.1   = localhost
+DNS.2   = 127.0.0.1
+DNS.2   = talent.test
+```
+
+Then run
+
+```
+sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout localhost.key -out localhost.crt -config localhost.conf
+```
+
+Enter in the appropriate information as prompted (should default to the values found in the above config)
+
+```
+sudo cp localhost.crt /etc/ssl/certs/localhost.crt
+sudo cp localhost.key /etc/ssl/private/localhost.key
+sudo apt install libnss3-tools
+certutil -d sql:$HOME/.pki/nssdb -A -t "P,," -n "localhost" -i localhost.crt
+```
+
+### Move files to be hosted
+
+#### Setup TalentCloud directory to be hosted
+
+```
+cd TalentCloud
+npm install
+npm run dev
+php artisan key:generate
+composer install
+php artisan migrate -n
```

Comment:
I'd adjust this slightly to
```bash
php artisan migrate:fresh
php artisan db:seed
```
Or the one-liner I used on branch changes:
```bash
composer install && npm install && npm run dev && php artisan migrate:fresh && php artisan db:seed
```

Reply:
Will do.

Reply:
@Khristophor 
Made the requested changes